File: LC-1295/LC-1295.ts
/**
 * Concepts: Stringify
 */

function findNumbers(nums: number[]): number {
    let evenDigitNum: number = 0, numsLen: number = nums.length;
    
    for (let i: number = 0; i < numsLen; ++i) {
        let numStr: string = String(nums[i]);
        if (numStr.length % 2 === 0) ++evenDigitNum
    }
    return evenDigitNum
};

File: LC-1295/LC-1295.cpp
#include <string>
#include <vector>

using namespace std;

class Solution {
public:
    /**
     * Concepts: Stringify
     */
    int findNumbers(vector<int>& nums) {
        int evenDigitNum = 0, numsLen = nums.size();
        for (int i = 0; i < numsLen; ++i) {
            string numStr = to_string(nums[i]);
            if (numStr.size() % 2 == 0)
                ++evenDigitNum;
        }
        return evenDigitNum;
    }
};